internal/generator/utils.go

```diff
@@ -65,32 +65,37 @@ func isPublic(fieldName string) bool {
 	return char != utf8.RuneError && unicode.IsUpper(char)
 }
 
-func checkDefaultValue(fieldType string, tag string) error {
+func checkDefaultValue(fieldType string, tag string) (any, error) {
 	var err error
-
+	var actualValue any
 	switch fieldType {
 	case "int", "int8", "int16", "int32", "int64":
-		_, err = strconv.ParseInt(tag, 10, 64)
+		actualValue, err = strconv.ParseInt(tag, 10, 64)
 
 	case "uint", "uint8", "uint16", "uint32", "uint64":
-		_, err = strconv.ParseUint(tag, 10, 64)
+		actualValue, err = strconv.ParseUint(tag, 10, 64)
 
 	case "float32", "float64":
-		_, err = strconv.ParseFloat(tag, 64)
+		actualValue, err = strconv.ParseFloat(tag, 64)
 
 	case "time.Duration":
 		_, err = time.ParseDuration(tag)
+		actualValue = tag
+
+	case "bool":
+		actualValue, err = strconv.ParseBool(tag)
```

Comment:
I suggest not using the `ParseBool` function, because in that case we are providing an interface to our users that is not transparent.

This means that clients should have a simple possible interface for setting their `Options` by specifying only the value `true` or `false`.

Reply:
i fixed it, now only supports true/false